.env.example

```diff
@@ -0,0 +1,21 @@
+# Core Variables (required for all samples)
+AM_URL=$AM_URL
+REALM_PATH=$REALM_PATH
+SCOPE=$SCOPE
+TIMEOUT=$TIMEOUT
+WEB_OAUTH_CLIENT=$WEB_OAUTH_CLIENT
+
+# Additional for Central & Embedded Login
+REDIRECT_URI=$REDIRECT_URI
+TREE=$TREE                          # Embedded Login Only
+
+# Additional Variables (for Angular & React Samples)
```

Comment:
Non-blocking: Can we make these comment the same grammatically as the one on line 8?